api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessConstants.java

```diff
@@ -13,4 +13,5 @@
 public class LocationBasedAccessConstants {
 
     public static final  String LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME = "locationbasedaccess.locationAttributeUuid";
+    public static final String DAEMON_USER_UUID = "A4F30A1B-5EB9-11DF-A648-37A07F9C90FB";
```

Comment:
Instead of duplicating this which is already in the Daemon class, can't we just use Daemon.isDaemonUser()?

Reply:
In the Daemon class, it was protected. So Can't use in outside. 
In AppUI and EMRAPI also, we have defined this as a separate constant.

Reply:
Did you understand my alternative suggestion?

Reply:
Ya Ya, Got it now and changed the code.

Reply:
Have you tested again to confirm that all is well?